flipperdevices/flipperzero-firmware
# [FL-1239] CLI commands readable list

# What's new

Change cli commands container to sorted tree

# Verification 

- Build and upload to f5 target
- Verify correct CLI commands execution
- Verify that help / ? CLI commands show 2 columns of available commands in alphabetical order. 
- Check for odd and even number of commands by launching iButton and Infrared apps

# Checklist (do not modify)
